src/artm/regularizer_sandbox/smooth_sparse_phi.cc

```diff
@@ -53,55 +37,36 @@ bool SmoothSparsePhi::RegularizePhi(::artm::core::Regularizable* topic_model, do
     has_dictionary = false;
   }
 
+  std::map<core::Token, float> coeffs;
```

Comment:
Same questions as for label_regularizer.cc:
- avoid coeffs variable and find the coeff "on fly"?
- handle gracefully user dictionary with missing tokens
- move expensive stuff out from the inner loop:
--- core::is_member(token.class_id, config_.class_id()
--- coeffs.find(token)